protocol/localstatequery/queries.go

```diff
@@ -597,8 +597,7 @@ func (u *UtxoId) MarshalCBOR() ([]byte, error) {
 	return cbor.Encode(tmpData)
 }
 
-// TODO (#863)
-type DebugEpochStateResult any
+type DebugEpochStateResult cbor.RawMessage
```

Comment:
This isn't the actual shape, so we should not remove the TODO (or mark that issue closed) as this is functionally no different than any.